README.md

```diff
@@ -29,8 +30,11 @@ reasonable values step by step. Feeling doubtful about what to type? Don't bothe
 Fire a command as is, and allowed values for first yet to be specified argument are to be
 handed to you after a while.
 
+`version` can be any valid semantic version select expressions. E.g: `5`, `5.15`, `5.15.2`, `>=5.9,<5.10`, ... Or just use `latest` to match the latest stable qt version(`5.15.2` until now).
```

Comment:
I believe this shall go into the help of the version option. Also, I would avoid clause _`5.15.2` until now_ anyway, since it puts us in a position when we have to update this value each time a new version is delivered.

Reply:
Of course, `--help` also modified. The `version` option can support not only `latest`.

Reply:
By the way, what is the real use case for the notation like `>=5.9,<5.10`?

Reply:
It's only rewards. Because I use `SimpleSpec.select()` expressions, and that's a valid expression: https://pypi.org/project/semantic-version/#using-a-specification